salt/modules/openstack_mng.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+'''
+Module for OpenStack Management
+
+:codeauthor:    Konrad Mosoń <mosonkonrad@gmail.com>
+:maturity:      new
+:depends:       openstack-utils
+:platform:      linux
+'''
+
+from __future__ import absolute_import
+
+# Import python libs
+import logging
+
+# Import salt libs
+
+# Import third party libs
+import os.path
+if os.path.isfile('/usr/bin/openstack-service'):
+    HAS_OPENSTACK = True
+else:
+    HAS_OPENSTACK = False
+
+log = logging.getLogger(__name__)
+
+# Define the module's virtual name
+__virtualname__ = 'openstack_mng'
+
+
+def __virtual__():
+    '''
+    Only load this module if openstack-service is installed
+    '''
+    if HAS_OPENSTACK:
```

Comment:
We can just replace the block above with `os.path.isfile('/usr/bin/openstack-service')` here and reduce the code a bit.